source/devices/AM65X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -350,8 +350,7 @@ Composer Studio.
 Host Support
 ============
 
-The Processor SDK is developed, built and verified on Ubuntu 16.04 and 18.04. Details on how to create a virtual machine to load Ubuntu
-are described in :ref:`this page <how-to-build-a-ubuntu-linux-host-under-vmware>`.
+The Processor SDK is developed, built and verified on Ubuntu 22.04.
```

Comment:
@jeevantelukula  Better to use the macro in all places. 

```suggestion
The Processor SDK is developed, built and verified on  Ubuntu |__LINUX_UBUNTU_VERSION_SHORT__|.
```

Note - You might also have to add `.. include:: /_replacevars.rst` 

Check - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/source/linux/Overview/Processor_SDK_Linux_Getting_Started_Guide.rst?plain=1#L1

Reply:
When built locally, it shows the accurate version even without including `replacevars.txt`

Reply:
This was done for `source/linux/Overview/Processor_SDK_Linux_Getting_Started_Guide.rst`, but not for any of the device specific release notes. You should use `|__LINUX_UBUNTU_VERSION_SHORT__|` in those instances as well.